common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -507,6 +504,25 @@ private ServiceException getServiceException(@NonNull final BrokerResult brokerR
 
     }
 
+    /**
+     * Helper method to retrieve UiRequiredException from BrokerResult
+     *
+     * @return {@link com.microsoft.identity.common.java.exception.UiRequiredException}
+     */
+    @NonNull
+    private UiRequiredException getUiRequiredException(@NonNull final BrokerResult brokerResult) {
+        final String errorCode = brokerResult.getErrorCode();
+        final UiRequiredException exception = new UiRequiredException(
+                brokerResult.getErrorCode(),
```

Comment:
>brokerResult.getErrorCode() [](http://example.com/codeflow?start=16&length=27)

Can be replaced with errorCode you just created.

Reply:
Done!